src/engraving/dom/slur.h

```diff
@@ -122,13 +122,18 @@ class Slur final : public SlurTie
 
     void undoSetIncoming(bool incoming);
     void undoSetOutgoing(bool outgoing);
+    void setIncoming(bool incoming);
+    void setOutgoing(bool outgoing);
     bool isIncoming() const;
     bool isOutgoing() const;
 private:
     M_PROPERTY2(int, sourceStemArrangement, setSourceStemArrangement, -1)
     M_PROPERTY2(ConnectedElement, connectedElement, setConnectedElement, ConnectedElement::NONE)
     M_PROPERTY2(PartialSpannerDirection, partialSpannerDirection, setPartialSpannerDirection, PartialSpannerDirection::NONE)
 
+    PartialSpannerDirection setIncomingCalcDirection(bool incoming);
+    PartialSpannerDirection setOutgoingCalcDirection(bool outgoing);
```

Comment:
The name of these methods is a little bit confusing; `set…` suggests that it is a setter function but it isn't.